### PR TITLE
setup doc

### LINKED ISSUE #863: Document the process of setting up raindex on a new chain
This needs to cover
- deploying all contracts including ob, arbs, dispair and subparsers
- ensuring there is a subgraph for the metaboard where the above are pushing meta to
- deploying a new subgraph for the ob deployed above
- setting up bot

### PR DESCRIPTION
**Pull request by rouzwelt**
## Motivation
addresses #863 

## Checks

By submitting this for review, I'm confirming I've done the following:
- [x] made this PR as small as possible
- [ ] unit-tested any new functionality
- [x] linked any relevant issues or PRs

### CODE REVIEW

**Comment by hardyjosh** [comment]
image

File: SETUP.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+```sh
+npx graph codegen
+npx graph build
+```
+and run the following command if you are using goldsky as your subgraph infra:
+```sh
+cd subgraph && goldsky subgraph deploy 'subgraph-name/subgraph-version'
+```
+
+For deploying the contracts locally you need to have foundry (either installed locally or enter nix shell that puts everything on your path) and run the following command:
```
Comment (thedavidmeister) [inline SETUP.md:15]: local deploys are not ok, not sure why this is here

**Review by thedavidmeister — CHANGES_REQUESTED** [review]

**Review by thedavidmeister — CHANGES_REQUESTED** [review]
i don't think all caps .md docs in the root of this repo are where process level documentation should be

who is this for? the docs should be where that person is going to find them

**Comment by dcatki** [comment]
@rouzwelt what's the status of this?

**Comment by rouzwelt** [comment]
moved to org-process repo

**Pull request closed without merging**